mlir/test/Integration/GPU/CUDA/indirect-call.mlir

```diff
@@ -0,0 +1,34 @@
+// RUN: mlir-opt %s \
+// RUN: | mlir-opt -gpu-lower-to-nvvm-pipeline="cubin-format=%gpu_compilation_format" \
+// RUN: | mlir-cpu-runner \
+// RUN:   --shared-libs=%mlir_cuda_runtime \
+// RUN:   --shared-libs=%mlir_runner_utils \
+// RUN:   --entry-point-result=void \
+// RUN: | FileCheck %s
+
+// CHECK: Hello from 0, 1, 3.000000
+module attributes {gpu.container_module} {
```

Comment:
Thanks for your work. 
I see that you added an integration test. However, we prefer using lit testing for several reasons: 1) it allows us to see the transformations happening in your code, and 2) we don't need to run the code, making it faster.

Reply:
Thanks for such a fast feedback. I have replaced the integration test by a test in `Dialect/GPU`.